snapshot.sh

```diff
@@ -11,6 +11,7 @@ SNAPSHOT_PATH="${SNAPSHOT_PATH}"
 SNAPSHOT_PREFIX="${SNAPSHOT_PREFIX:-$CHAIN_ID}"
 SNAPSHOT_RETAIN="${SNAPSHOT_RETAIN:-2 days}"
 SNAPSHOT_METADATA="${SNAPSHOT_METADATA:-1}"
+SNAPSHOT_FORMAT="${SNAPSHOT_FORMAT:-tar.gz}"
```

Comment:
Define all variables with defaults here
Update default value for `SNAPSHOT_FORMAT` based on `SNAPSHOT_COMPRESS`
```suggestion
SNAPSHOT_COMPRESS=${SNAPSHOT_COMPRESS:-1}
SNAPSHOT_FORMAT_DEFAULT=tar
if [ "$SNAPSHOT_COMPRESS" != "0" ]; then
  SNAPSHOT_FORMAT_DEFAULT=tar.gz
end
SNAPSHOT_FORMAT="${SNAPSHOT_FORMAT:-$SNAPSHOT_FORMAT_DEFAULT}"
```

Reply:
Thank you!